src/pages/en/core-concepts/astro-components.md

```diff
@@ -275,10 +275,15 @@ In Astro, you use the standard `kebab-case` format for all HTML attributes inste
 
 In Astro, you can use standard HTML comments where JSX would use JavaScript style comments.
 
-```html title="example.astro"
+```astro title="example.astro"
 <!-- HTML comment syntax is valid in .astro files -->
+{/* JS comment  */}
 ```
 
+:::caution
+The HTML comments will be included in browser DOM, while JS ones will be skipped. In case you need to leave TODO messages or other development-only explanations in JSX part, use JavaScript style comments instead.
+:::
```

Comment:
```suggestion
:::caution
HTML-style comments will be included in browser DOM, while JS ones will be skipped. To leave TODO messages or other development-only explanations, you may wish to use JavaScript-style comments instead.
:::
```

Reply:
Thanks for much better copy. Writing texta is not my strongest part :) will push changes accordingly soon! 👌

Reply:
Actually there is one edge case when even js comments could end up in the html, which is [is:raw](https://docs.astro.build/en/reference/directives-reference/#israw) directive, but I am pretty sure it is self-explanatory on it's own, so did not include in this docs section.

Reply:
Alright, let me take one more pass at it then, without making it too wordy....  😄